config/dev-realm-spsh.json

```diff
@@ -1237,6 +1238,60 @@
           "offline_access",
           "microprofile-jwt"
         ]
+      },
+      {
+        "id": "8d129be5-0cde-44e7-b9de-59797f650e4b",
+        "clientId": "${KC_NEXTCLOUD_CLIENT_ID}",
+        "name": "NextCloud-Dateiablage",
+        "description": "",
+        "rootUrl": "",
+        "adminUrl": "",
+        "baseUrl": "",
+        "surrogateAuthRequired": false,
+        "enabled": true,
+        "alwaysDisplayInConsole": false,
+        "clientAuthenticatorType": "client-secret",
+        "secret": "${KC_NEXTCLOUD_CLIENT_SECRET}",
+        "redirectUris": [
+          "https://sh.lumaserv.dev/*"
+        ],
+        "webOrigins": [
+          "/*"
+        ],
+        "notBefore": 0,
+        "bearerOnly": false,
+        "consentRequired": false,
+        "standardFlowEnabled": true,
+        "implicitFlowEnabled": true,
+        "directAccessGrantsEnabled": true,
```

Comment:
Disable the implicit flow (unless its needed) and definitely disable the direct access grants